app/views/agents/creneaux/agent_searches/index.html.slim

```diff
@@ -5,6 +5,8 @@
   .card-body
     h5.card-title Configuration des filtres
     = simple_form_for(Creneau::AgentSearch.new, remote: true, url: organisation_agent_searches_path(@organisation), method: :get) do |f|
+      - if @user
+        = hidden_field_tag :user, @user
```

Comment:
I think it would be really nice to make this visible to the agent. We could remind him with a sentence or a disabled input that he's currently searching a creneau for a specific user.

Reply:
also, as said before, please suffix with `_id`